docs/install.md

```diff
@@ -1,51 +1,73 @@
 # Installing Open Angel Arena
-You have to set up several things before you can run a local instance of AAA for development.
+You have to set up several things before you can run a local instance of OAA for development.
 
 ## System Dependencies
 Start by installing these programs:
  * [Steam](http://steampowered.com)
  * [nodejs](http://nodejs.org)
  * [Github client](http://desktop.github.com/)
- * [Sublime Text 3](http://www.sublimetext.com/) **optional**
```

Comment:
Don't remove ST3. A lot of people are using it.
And if you're at it add Atom Editor as well.